storage/client/src/main/java/com/azure/storage/queue/models/UpdatedMessage.java

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.storage.queue.models;
+
+import java.time.OffsetDateTime;
+
+public final class UpdatedMessage {
+    /*
+     * The pop receipt of the queue message.
+     */
+    private final String popReceipt;
+
+    /*
+     * A UTC date/time value that represents when the message will be visible
+     * on the queue.
+     */
+    private final OffsetDateTime timeNextVisible;
+
+    public UpdatedMessage(String popReceipt, OffsetDateTime timeNextVisible) {
+        this.popReceipt = popReceipt;
+        this.timeNextVisible = timeNextVisible;
+    }
+
+    public String popReceipt() {
+        return popReceipt;
+    }
+
+    public OffsetDateTime timeNextVisible() {
+        return timeNextVisible;
+    }
```

Comment:
This class API and the name of the class are not obviously related. Is `UpdatedMessage` really the right name here?